package.json

```diff
@@ -1,14 +1,22 @@
 {
   "name": "@nextcloud/auth",
-  "version": "1.3.0",
-  "description": "",
+  "version": "2.0.0-next",
+  "description": "Nextcloud helpers related to authentication and the current user",
```

Comment:
I would say it is authentication and authorization (*isAdmin*)

Reply:
I've just copied from package repo 🤣